### PR TITLE
[SPARK-45195][FOLLOWUP] Update python example

### PR DESCRIPTION
**Pull request by zhengruifeng**
To address https://github.com/apache/spark-website/pull/477#discussion_r1329098596

image

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -88,13 +88,17 @@ navigation:
...
                 <div class="code">
                     <p>$ pip install pyspark</p>
                     <p>$ pyspark</p>
-                    <p>$ </p>
+                </div>
+                <div style="font-size: 16px;">Try offical image
+                </div>
+                <div class="code">
                     <p>$ docker run -it --rm spark:python3 /opt/spark/bin/pyspark</p>
+                    <p>$ </p>
```
Comment (gatorsmile) [inline index.md]: remove this line?

File: index.md [hunk 2]
```diff
@@ -88,13 +88,17 @@ navigation:
...
                 <div class="mb-2 title">Run now</div>
-                <div style="font-size: 16px;">Install with 'pip' or try offical image
+                <div style="font-size: 16px;">Install with 'pip'
                 </div>
                 <div class="code">
                     <p>$ pip install pyspark</p>
                     <p>$ pyspark</p>
-                    <p>$ </p>
+                </div>
+                <div style="font-size: 16px;">Try offical image
```
Comment (gatorsmile) [inline index.md]: Typo: offical => official 

```
Use the official Docker image.
```

**Review by HyukjinKwon — APPROVED** [review]

**Review by yaooqinn — APPROVED** [review]

**Comment by zhengruifeng** [comment]
thank you guys

File: index.md [hunk 3]
```diff
@@ -88,12 +88,15 @@ navigation:
...
                 <div class="mb-2 title">Run now</div>
-                <div style="font-size: 16px;">Install with 'pip' or try offical image
+                <div style="font-size: 16px;">Install with 'pip'
                 </div>
                 <div class="code">
                     <p>$ pip install pyspark</p>
                     <p>$ pyspark</p>
-                    <p>$ </p>
+                </div>
+                <div style="font-size: 16px;">Use the official Docker image
```
Comment (allisonwang-db) [inline index.md:97]: super nit: it would be great to hyperlink this Docker image (if it is not mentioned anywhere else).

**Review by allisonwang-db — COMMENTED** [review]
Thanks for the update!

**Pull request merged**